requirements.txt

```diff
@@ -14,4 +14,5 @@ black
 pytest-mock
 scrapinghub
 scrapinghub-entrypoint-scrapy
-itemadapter
\ No newline at end of file
+itemadapter
+pre-commit
```

Comment:
I'm not sure how requirements.txt are organized, but probably it'd be better to make an explicit requirement-dev.txt file, as not all of these requirements are needed to run spidermon.

Also, it seems some of the requirements which are here are not in setup.py, e.g. itemadapter is not in setup.py install_requires (it's in test_requires for some reason).

Reply:
Yeah, the package requirements are weirdly structured, we could open a new issue to fix that

Reply:
Makes sense; we don't need to fix it in this PR.

Not having itemadapter in install_requires looks like a separate issue, I think it's a bug.